src/ruby/spec/generic/rpc_server_spec.rb

```diff
@@ -414,8 +414,8 @@ def an_rpc(req, call)
             stub = SlowStub.new(alt_host, :this_channel_is_insecure)
             begin
               stub.an_rpc(req)
-            rescue GRPC::BadStatus => e
-              bad_status_code = e.code
+            rescue GRPC::ResourceExhausted
+              one_failed_as_unavailable = true
```

Comment:
Something is definitely off with this merge. This line changes from `bad_status_code` to `one_failed_as_unavailable`, but the check on line 425 still looks at `bad_status_code`.